### PR TITLE
Inventory add updated_by

### PR DESCRIPTION
**Pull request by Ebed-meleck**
Add the required note after price change
Add the column updated_by for the last user updated item

@jniles Did I answer this #7404

### CODE REVIEW

File: client/src/modules/inventory/list/modals/search.modal.html [hunk 1]
```diff
@@ -105,6 +105,15 @@
             on-select-callback="ModalCtrl.onSelectTags(tags)">
           </bh-tag-select>
 
+          <!-- user -->
+          <bh-user-select
+            user-id="ModalCtrl.searchQueries.updated_by"
+            label="FORM.LABELS.LAST_USER_UPDATE"
+            require="false"
```
Comment (jmcameron) [inline client/src/modules/inventory/list/modals/search.modal.html]: This is an error:  it should be 'required'.

**Review by jmcameron — CHANGES_REQUESTED** [review]
The code looks good.

I will wait for the author to change 'require' to 'required' (for bhUserSelect) before testing.

File: client/src/modules/inventory/list/modals/search.modal.html [hunk 2]
```diff
@@ -105,6 +105,15 @@
             on-select-callback="ModalCtrl.onSelectTags(tags)">
           </bh-tag-select>
 
+          <!-- user -->
+          <bh-user-select
+            user-id="ModalCtrl.searchQueries.updated_by"
+            label="FORM.LABELS.LAST_USER_UPDATE"
+            required="false"
```
Comment (jmcameron) [inline client/src/modules/inventory/list/modals/search.modal.html:112]: 👍

**Review by jmcameron — CHANGES_REQUESTED** [review]
The code looks good. 
The tests pass.
I  tested this on my dev computer with bhima_test and it works correctly.

I noticed that two English translation terms are missing:
image

Note that it is easy to check for missing translation terms by running: `npm run check:lang`

Once the translation item is fixed, this PR is ready to merge.

File: client/src/i18n/en/form.json [hunk 3]
```diff
@@ -581,6 +581,8 @@
             "LANGUAGE": "Language",
             "LAST_ENTRY_DATE": "Last Entry Date",
             "LAST_NAME": "Last Name",
+            "LAST_USER_UPDATE" : "Last edited by",
```
Comment (jmcameron) [inline client/src/i18n/en/form.json:584]: Thanks

**Review by jmcameron — APPROVED** [review]
Looks good.

**Comment by Ebed-meleck** [comment]
Closed #7404

**Pull request merged**